<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCSettinRule</class>
 <widget class="QWidget" name="QCSettinRule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1511</width>
    <height>947</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,10">
   <item>
    <widget class="CusTableView" name="QCProjectsTable" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="10,3,1">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
          </property>
          <property name="text">
           <string>    1-2s:1个质控点超出±2SD</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
          </property>
          <property name="text">
           <string>    R-4s:在同一批内高和低质控测定值之间的差值超过4s</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
          </property>
          <property name="text">
           <string>    1-3s:1个质控点超出±3SD</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
          </property>
          <property name="text">
           <string>      4-1s:4个连续的质控测定值同时超过X-1s或X+1s</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox_6">
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
          </property>
          <property name="text">
           <string>   2-2s:2个连续的质控点超出±2SD</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="checkBox_5">
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
          </property>
          <property name="text">
           <string>    10-X:10个连续的质控测定值落在平均数()的同一侧</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>联合质控判断</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>质控品</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>目标值</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_9">
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>标准差</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>X</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>待显示...</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>待显示...</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>Y</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>待显示...</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>待显示...</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="CusAdjustBox" name="XAdjust">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="CusAdjustBox" name="YAdjust">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="SaveBtn">
          <property name="minimumSize">
           <size>
            <width>145</width>
            <height>45</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnNomal.png);
padding: -1;}
QPushButton:pressed{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnClicked.png);
padding: -1;}
</string>
          </property>
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="returnBtn">
          <property name="minimumSize">
           <size>
            <width>145</width>
            <height>45</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnNomal.png);
padding: -1;}
QPushButton:pressed{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnClicked.png);
padding: -1;}
</string>
          </property>
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CusTableView</class>
   <extends>QWidget</extends>
   <header location="global">CusTableView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CusAdjustBox</class>
   <extends>QFrame</extends>
   <header location="global">CusAdjustBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
